.github/workflows/stale_prs_and_issues.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue will be closed in 2 weeks in case of no activity.'
+          stale-pr-message: 'This PR will be closed in 2 weeks in case of no activity.'
+          close-issue-message: 'This issue was closed because it has been stalled for 1 week with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 1 week with no activity.'
+          days-before-stale: 14
+          days-before-close: 7
```

Comment:
Do we really need to close issues? What is expected within 2 weeks? If nobody fixes the issue, it does not mean it should be closed.

Reply:
Could you add this comment to original Jira https://jira.devtools.intel.com/browse/CVS-106776?